### ISSUE TITLE
Theme Colors + Accessibility

### ISSUE THREAD

**Issue by catarak** [processing/p5.js-web-editor#1263]
#### Nature of issue?

- Existing feature enhancement

#### Feature enhancement details:

There are many open issues about Accessibility and Contrast (see the [Accessibility Column](https://github.com/processing/p5.js-web-editor/projects/4#column-7404187)). Rather than address all of these individually, I think it makes sense to start by looking at the [design system](https://www.figma.com/file/cyHiandYyEAPj0Kj6tJbHc/p5-design-system-2018?node-id=0%3A937) and adjust the colors that haven't been updated. Maybe the light and dark themes should be WCAG AA, and the high contrast theme should be WCAG AAA? 

To make changes to these colors, one needs to make changes to the [theme variables](https://github.com/processing/p5.js-web-editor/blob/master/client/styles/abstracts/_variables.scss), which will update throughout the whole app.

To make changes to the syntax highlighting colors, one needs to change:
- [Light theme](https://github.com/processing/p5.js-web-editor/blob/master/client/styles/components/_p5-light-codemirror-theme.scss)
- [Dark theme](https://github.com/processing/p5.js-web-editor/blob/master/client/styles/components/_p5-dark-codemirror-theme.scss)
- [High contrast theme](https://github.com/processing/p5.js-web-editor/blob/master/client/styles/components/_p5-contrast-codemirror-theme.scss)

This issue does not need to be completed in one big PR, rather, it can be done incrementally.

**Comment by meganindya** [comment]
@catarak Can I work on this? I'd want to begin with the light theme.

**Comment by catarak** [comment]
In looking at #1342, I think there are also some CSS specificity issues!

**Comment by elit-altum** [comment]
@catarak Can I work on the high contrast theme ?
